store/clientSettings.ts

```diff
@@ -2,48 +2,79 @@ import { MutationTree, ActionTree } from 'vuex';
 import nuxtConfig from '~/nuxt.config';
 
 /**
- * Casted typings for the CustomPrefs property of DisplayPreferencesDto
+ * Mutations that should be synced to the server
  */
-export interface CustomPreferences {
+export const syncedMutations = ['SET_DARK_MODE', 'SET_LOCALE'];
```

Comment:
Why? Imo what shouldn't be synced should live in the root state.

This is just another thing we need to change and maintain when adding properties that we don't know if we will ever use. If it isn't useful now, I think it's better not to do it.

Reply:
For the first part, see the comment bellow this one.

For the second, not everything would make sense to be synced. For example, the theme might need to be device-dependent. 

This allows us to keep logical groupings of options in the same store, while allowing to sync some of them.